src/typescript/frontend/src/components/trade/DepositWithdrawModal/SelectMarketContent.tsx

```diff
@@ -104,13 +104,13 @@ export const SelectMarketContent: React.FC<{
     getCoreRowModel: getCoreRowModel(),
   });
   return (
-    <div className="flex w-full flex-col items-center">
+    <div className="flex max-h-[655px] min-h-[655px] w-full flex-col items-center overflow-y-hidden">
```

Comment:
Is there a reason you need to set min and max here, and not simply set the height to 655px?

Reply:
some weird css behavior when a max and min is not set